coalib/core/DependencyTracker.py

```diff
@@ -293,7 +293,7 @@ def check_circular_dependencies(self):
          ...
         coalib.core.CircularDependencyError.CircularDependencyError: ...
 
-        :raises coalib.core.CircularDependencyError.CircularDependencyError:
+        :raises CircularDependencyError:
```

Comment:
I made it fully qualified, due to an issue with sphinx 1.5.5 (see #4180) a few days ago. With this PR we no longer need the 'workaround' from #4180. Do you really want me to make it the name fully qualified again?